custom_components/tuya_local/devices/unknow_2way_energymeter.yaml

```diff
@@ -0,0 +1,452 @@
+# Can´t put some DPs to work (calibration and alarms)
+name: Bidirectional energy meter
+products:
+  - id: 8bjnabt4gh8tu4vr
+    name: Smart meters
+primary_entity:
+  entity: sensor
+  name: Forward energy total
+  class: energy
+  dps:
+    - id: 1
+      type: integer
+      optional: true
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      mapping:
+        - scale: 100
+secondary_entities:
+  - entity: sensor
```

Comment:
I think rather than being a separate sensor, these "hour" variants would be better as attributes on the overall sensor. With the class "total_increasing", they will appear to show the same thing in the UI, and the energy dashboard has ways to get periodic usage already.